content/hi/docs/setup/learning-environment/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+शीर्षक:  सीखने का माहौल
+वजन: 20
+---
+
+<!--
+{{/* There is a Netlify redirect from this page to /docs/tasks/tools/ */}}
+{{/* This page content only exists to provide a navigation stub */}}
+{{/* and to protect in case that redirect is one day removed. */}}
+
+{{/* If you're localizing this page, you only need to copy the front matter */}}
+{{/* and add a redirect into "/static/_redirects", for YOUR localization. */}}
+-->
+
+## kind
+
+[`kind`](https://kind.sigs.k8s.io/docs/) आपको Kकुबेरनेट्स को अपने स्थानीय कंप्यूटर पर चलाने देता है। इस
+ उपकरण के लिए आवश्यक है कि आपके पास 
+[Docker](https://docs.docker.com/get-docker/)स्थापित और कॉन्फ़िगर किया गया।
+
+kind [Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start)पृष्ठ
+आपको दिखाता है कि आपको काम करने के लिए क्या करना हैक्या करना हे
+
+## minikube
+
+इसलिए `kind`, [`minikube`](https://minikube.sigs.k8s.io/) iएक उपकरण है जो आपको कुबेरनेट्स चलाने देता है
+स्थानीय स्तर पर `।` minikube`c`आपके व्यक्तिगत पर एकल-नोड कुबेरनेट्स क्लस्टर चलाता है
+कंप्यूटर (विंडोज़, मैक और लिनक्स पीसी सहित) ताकि आपताकि आप कोशिश कर सकें
+कुबेरनेट्स, या दैनिक विकास कार्य के लिए।
+
+
+आप अधिकारी का अनुसरण कर सकते हैं 
+[शुरू हो जाओ!] (https://minikube.sigs.k8s.io/docs/start/) gमार्गदर्शन करें यदि आपका ध्यान है
+उपकरण स्थापित होने पर।पर। 
```

Comment:
```suggestion
```

Reply:
Can you add a redirect to `/static/_redirect` as mentioned by @sftim in previous comment